### ISSUE TITLE
Support contract addresses

### ISSUE THREAD

**Issue by kalepail** [stellar/friendbot#6]
### What problem does your feature solve?
Contracts can hold value and with the rise of smart wallets being able to airdrop funds to contracts will be increasingly important.

### What would you like to see?
e.g. `rpc.requestAirdrop(contractId)` should work vs:
```json
{
  "type": "https://stellar.org/horizon-errors/bad_request",
  "title": "Bad Request",
  "status": 400,
  "detail": "The request you sent was invalid in some way.",
  "extras": {
    "invalid_field": "addr",
    "reason": "invalid version byte"
  }
}
```

### What alternatives are there?
You have to fund a G-address and then use the SAC interface to move funds from the G-address to your C-address. It's a bit cumbersome.

**Comment by tomerweller** [comment]
> However it’s unclear to me how valuable/priority that is, because the main reason folks need the native asset on testnet is to pay fees. Contracts can’t pay fees. Therefore contracts have much less use for native.

@leighmcculloch I don't agree with this statement. It's true that XLM provide a utility for paying fees, but they are also the canonical asset of the network and are the go-to asset for testing the network's functionality.

**Comment by tomerweller** [comment]
Why is this dependent on rpc? can't we use the no-simulation sac transfer option in the sdk?
